OvmfPkg/VirtMmCommunicationDxe/VirtMmCommunication.inf

```diff
@@ -13,6 +13,7 @@
   MODULE_TYPE                    = DXE_RUNTIME_DRIVER
   VERSION_STRING                 = 1.0
   ENTRY_POINT                    = VirtMmCommunication2Initialize
+  VALID_ARCHITECTURES            = X64 AARCH64

```

Comment:
NIT: Shouldn't RISC-V also be added here now?

Reply:
It's done by another patch in the series which also updates other sections in the file for riscv